src/components/asset-list/RecyclerAssetList2/FastComponents/FastBalanceCoinRow.tsx

```diff
@@ -110,28 +123,42 @@ const MemoizedBalanceCoinRow = React.memo(
                     {item?.name}
                   </Text>
                 </View>
-
-                <Text
+                <LiveTokenText
+                  selector={tokenBalanceSelector}
+                  tokenId={uniqueId}
+                  initialValueLastUpdated={priceUpdatedAt}
+                  initialValue={item?.native?.balance?.display ?? `${nativeCurrencySymbol}0.00`}
+                  autoSubscriptionEnabled={false}
+                  color={{ custom: theme.colors.dark }}
+                  size={'16px / 22px (Deprecated)'}
+                  weight="bold"
                   align="right"
-                  color={{ custom: valueColor }}
-                  size="16px / 22px (Deprecated)"
-                  weight="medium"
-                  testID={`balance-coin-row-value-${item?.name}`}
-                >
-                  {item?.native?.balance?.display ?? `${nativeCurrencySymbol}0.00`}
-                </Text>
+                />
               </View>
 
               <View style={[sx.row, sx.bottom]}>
                 <View style={sx.textWrapper}>
-                  <Text color={{ custom: theme.colors.blueGreyDark50 }} numberOfLines={1} size="14px / 19px (Deprecated)" weight="medium">
-                    {nativeDisplay ?? ''}
+                  <Text color={{ custom: theme.colors.blueGreyDark50 }} numberOfLines={1} size="14px / 19px (Deprecated)" weight="bold">
+                    {nativeBalanceDisplay}
                   </Text>
                 </View>
-
-                <Text align="right" color={{ custom: changeColor }} size="14px / 19px (Deprecated)" weight="medium">
-                  {percentageChangeDisplay}
-                </Text>
+                <LiveTokenText
+                  selector={tokenPriceChangeSelector}
+                  tokenId={uniqueId}
+                  initialValueLastUpdated={priceUpdatedAt}
+                  initialValue={formatPercentChange(percentChange)}
+                  autoSubscriptionEnabled={false}
+                  usePriceChangeColor
+                  priceChangeChangeColors={{
+                    positive: theme.colors.green,
+                    negative: theme.colors.blueGreyDark50,
+                    neutral: theme.colors.blueGreyDark50,
+                  }}
```

Comment:
Discriminate prop types to not allow you to specify the colors if it's not enabled